aws/README.md

```diff
@@ -0,0 +1,174 @@
+# linkerd on AWS ECS with Consul
+
+This example demonstrates deploying linkerd on an AWS ECS cluster, using Consul
+for service discovery.
+
+The instructions below assume you have already configured AWS with the proper
+IAM, key pairs, and VPCs for an ECS cluster. For more information start here:
+http://docs.aws.amazon.com/AmazonECS/latest/developerguide/get-set-up-for-amazon-ecs.html
+
+## Overview
+
+The following components make up the system:
+* `ECS`: Docker container management. Every ECS instance runs the following
```

Comment:
I think we should give a little bit more background here explaining that ECS doesn't have a 1-per-node notion and so to get 1-per-node we have to run these outside of the ECS scheduler and launch them directly through the launch configuration.